StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -198,12 +198,27 @@ extension PaymentSheet {
                 )
             // MARK: ↪ Deferred Intent
             case .deferredIntent(let intentConfig):
-                handleDeferredIntentConfirmation(
-                    confirmType: .new(
+                let confirmType: ConfirmPaymentMethodType? = if let bank = confirmParams.instantDebitsLinkedBank {
```

Comment:
Can you add a comment explaining this a bit? IDK maybe something like "Instant Debits should be handled as if it were a saved payment method because x"

Reply:
Also instead of this hacky thing where we're using `Paymentoption.new` but then overriding logic to make it confirm like it's saved, did you consider other options like
1. handling instant debits as a `PaymentOption.saved` 
2. making a new `PaymentOption.instantDebits` type

Reply:
I went with option 1 and created a `PaymentOption.saved` in `PaymentMethodFormViewController.paymentOption`. What do you think?